src/app.rs

```diff
@@ -213,76 +209,11 @@ impl App {
             return Err(ServerError::IdentityAlreadyDeleted);
         }
 
-        // Check if the id is already queued for deletion
-        if tx.identity_is_queued_for_deletion(commitment).await? {
```

Comment:
the `insert_new_deletion` method has been changed to be idempotent. Therefore there's no longer a need to check if we have already accepted a deletion